### ISSUE TITLE
re: hour_logger

### ISSUE THREAD

**Issue by northwinds** [matryer/bitbar-plugins#1164]
FAO: (@udeyrishi)
Hi there
I just tried to install your BitBar plugin and I get a yellow exclamation mark that when i click on shows: 
"[path to my BitBar plugins folder]/hours.1m.sh: line 14:  usr/local/bin/hours: No Such File or Directory "
Any ideas how I fix this?

Many thanks!

**Comment by udeyrishi** [comment]
Hey @northwinds 👋 Had to dust this one off the shelf, hadn't touched it in 3 years 😅. 

The way I wrote this originally, this plugin was an extension to a command line script that I hacked together a while ago: https://github.com/udeyrishi/hour_logger. You'll need to install that first; see the readme in the repo. 

It's possible that this plugin can be made its own standalone thing. PRs are welcome 😉

**Comment by northwinds** [comment]
Fantastic!
Thanks for the speedy reply!

After a bunch of fiddling I’ve finally got it working!

Now I just need to find some work to track! ;-)

One question though, I’m in the UK, is it possible to change the BitBar plugin from $ to £ ?

I thought it must be the modes.py file that controlled this but when I searched for “$” and replaced the 5 instances I found with “£” then restarted, the $ symbol remained?


Many thanks

Craig Duncan.

On 17 January 2019 at 01:47:41, Udey Rishi (notifications@github.com) wrote:

Hey @northwinds 👋 Had to dust this one off the shelf, hadn't touched it in 3 years 😅.

The way I wrote this originally, this plugin was an extension to a command line script that I hacked together a while ago: https://github.com/udeyrishi/hour_logger. You'll need to install that first; see the readme in the repo.

It's possible that this plugin can be made its own standalone thing. PRs are welcome 😉
